README.md

```diff
@@ -1,2 +1,50 @@
-# AwesomeBooks-Module
-Microverse project for practicing implementing modules (javascript)
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# AwesomeBooks-Project
+Awesome books: plain JavaScript with objects
+
+## Acces link
+
+> https://vitorguedesmadeira.github.io/AwesomeBooks-Project/
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- LocalStorage
```

Comment:
- [ ] In your README kindly include instructions on setting up and running the app locally. i.e. from cloning to the actual running of the app.